documentation/docs/backend/allowDynamicBackends.mdx

```diff
@@ -0,0 +1,32 @@
+---
+hide_title: false
+hide_table_of_contents: false
+pagination_next: null
+pagination_prev: null
+---
+import {Fiddle} from '@site/src/components/fiddle';
+
+# allowDynamicBackends
+
+The **`allowDynamicBackends()`** function is used to control whether or not Dynamic Backends should be allowed within this Fastly Compute Service.
+
+By default, Dynamic Backends are enabled, but can be a potential security concern since third-party JavaScript code may send arbitrary requests, potentially including sensitive/secret data, off to destinations that the JavaScript project was not intending.
+
+Using `allowDynamicBackends(false)` this security property can be restored to only use explicit backend definitions.
+
+>**Note**: Dynamic Backends are disabled by default for Fastly Services. Please contact [Fastly Support](https://support.fastly.com/hc/requests/new?ticket_form_id=360000269711) to request the feature be enabled or disabled on Fastly Services.
```

Comment:
The wording is a little confusing here wrt above. To someone who doesn't grasp the difference between the two levels of enable, "by default, enabled ... disabled by default" could read as a contradiction.

Maybe we could say "By default, Dynamic Backends are enabled in the SDK, but disabled at the Fastly service configuration level. This function allows disabling at the SDK level for security reasons [details]. In order to use dynamic backends, the default setting in the SDK suffices, but the feature must be enabled at the Fastly service level. Please contact support ..."

Reply:
Thanks I've revised the page here.